From 10042d803ed44d826e4ce231c86dbc63a8ddcf08 Mon Sep 17 00:00:00 2001
From: Eric Laurent <elaurent@google.com>
Date: Thu, 23 Feb 2023 21:05:47 +0000
Subject: [PATCH] Revert "audio policy: fix sensor privacy listener"

This reverts commit 60aa4d06161f40e9f18c44caf0fc057b86deceb9.

Bug: 270570809
Bug: 258021433
Reason for revert: b/270570809
(cherry picked from https://googleplex-android-review.googlesource.com/q/commit:6f0e8e89aeb69ef8cd9bc6ed85cd3a4af7cd11b4)
Merged-In: I6e17d5d7a598fb01f3bf75f2df88d63d187231d9
Change-Id: I6e17d5d7a598fb01f3bf75f2df88d63d187231d9
---
 services/audiopolicy/service/AudioPolicyService.cpp | 2 --
 1 file changed, 2 deletions(-)

diff --git a/services/audiopolicy/service/AudioPolicyService.cpp b/services/audiopolicy/service/AudioPolicyService.cpp
index bfce4bab01..e7d945fad6 100644
--- a/services/audiopolicy/service/AudioPolicyService.cpp
+++ b/services/audiopolicy/service/AudioPolicyService.cpp
@@ -1803,14 +1803,12 @@ void AudioPolicyService::UidPolicy::dumpInternals(int fd) {
 void AudioPolicyService::SensorPrivacyPolicy::registerSelf() {
     SensorPrivacyManager spm;
     mSensorPrivacyEnabled = spm.isSensorPrivacyEnabled();
-    (void)spm.addToggleSensorPrivacyListener(this);
     spm.addSensorPrivacyListener(this);
 }
 
 void AudioPolicyService::SensorPrivacyPolicy::unregisterSelf() {
     SensorPrivacyManager spm;
     spm.removeSensorPrivacyListener(this);
-    spm.removeToggleSensorPrivacyListener(this);
 }
 
 bool AudioPolicyService::SensorPrivacyPolicy::isSensorPrivacyEnabled() {
-- 
2.41.0.162.gfafddb0af9-goog

